presto-oracle/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>326-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-oracle</artifactId>
+    <description>Presto - Oracle Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <oracle.version>19.3.0.0</oracle.version>
```

Comment:
Inline this property since it's only used in one place. We normally only use properties when there are multiple related dependencies that need the same version.

Reply:
done